cli/azd/resources/scaffold/templates/host-containerapp.bicept

```diff
@@ -293,11 +330,11 @@ resource linkerCreatorRole 'Microsoft.Authorization/roleAssignments@2022-04-01'
   }
 }
 {{- end}}
-{{- if .DbMySql}}
+{{- if (and .DbPostgres .DbPostgres.AuthUsingManagedIdentity)}}
```

Comment:
FYI: I changed the order of MySQL and PostgreSQL: Always keep PostgreSQL on top of MySQL in all related fragments.